docs/api/nav.md

```diff
@@ -24,19 +21,31 @@ import APITOCInline from '@components/page/api/APITOCInline';
 
 <EncapsulationPill type="shadow" />
 
-<h2 className="table-of-contents__title">Contents</h2>
+Nav is a standalone component for loading arbitrary components and pushing new components on to the stack.
 
-<APITOCInline
-  toc={toc}
-  maxHeadingLevel={2}
-  autogenerated={[Props, Events, Methods, Parts, CustomProps, Slots]}
-/>
+Unlike Router Outlet, Nav is not tied to a particular router. This means that if we load a Nav component, and push other components to the stack, they will not affect the app's overall router. This fits use cases where you could have a modal, which needs its own sub-navigation, without making it tied to the apps URL.
 
+## Using NavLink
 
+NavLink is a simplified API when interacting with Nav. Developers can customize the component, pass along component properties, modify the direction of the route animation or define a custom animation when navigating. 
 
-Nav is a standalone component for loading arbitrary components and pushing new components on to the stack.
+import NavLinkExample from '@site/static/usage/nav/nav-link/index.md';
 
-Unlike Router Outlet, Nav is not tied to a particular router. This means that if we load a Nav component, and push other components to the stack, they will not affect the app's overall router. This fits use cases where you could have a modal, which needs its own sub-navigation, without making it tied to the apps URL.
+<NavLinkExample />
+
+## Navigation within a Modal
```

Comment:
It looks like the button is partially cut off in the Angular example: 
![image](https://user-images.githubusercontent.com/2721089/186960228-17fce7c5-b83e-46cb-9d62-a68d9d6f3d80.png)

Reply:
In React and Vue, the usage of `ion-app` creates a container element around the children content, creating a DOM tree like:

```html
<ion-app class="ion-page">
  <div class="ion-page">
    <!-- component here -->
  </div>
</ion-app>
```

Angular does not do the same thing (likely we never noticed/wasn't an issue because `ion-router-outlet` handles this implementation detail. 

```html
<ion-app class="ion-page">
  <!-- component here -->
</ion-app>
```

Should I track this as a bug in Ionic or would we rather update our Playground demo code to workaround this issue (e.g.):
```html
<ion-app>
  <app-example class="ion-page"></app-example>
</ion-app>
```

Developers using `ion-router-outlet` should never run into this issue.